src/property.ts

```diff
@@ -90,7 +93,8 @@ export const DEFAULT_PROPERTY_PRECEDENCE: Property[] =  [
 
   // Nested Encoding Property
   Property.SCALE_TYPE,
-  Property.SCALE_ZERO
+  Property.SCALE_ZERO,
+  Property.SCALE_ROUND,
```

Comment:
Why Round after ZERO here?  

Unless you have convincing reason to do so, always use alphabetical order for readability.  (You actually do it else where in this PR)